integration_tests/e2e/selectGender.cy.ts

```diff
@@ -69,4 +69,20 @@ context('Select Gender', () => {
       },
     )
   })
+
+  it(`Back link goes back to manage contact`, () => {
+    cy.task('stubGetGenders')
+
+    Page.verifyOnPage(ManageContactDetailsPage).clickChangeGenderLink()
+    Page.verifyOnPage(SelectGenderPage, 'Jones Mason') //
+      .backTo(ManageContactDetailsPage, 'Jones Mason')
+  })
+
+  it(`Cancel goes back to manage contact`, () => {
```

Comment:
Is this going to overlap with Rosario's integration tests for gender? Sure we can sort in merge anyway.

Reply:
I think his is just adding it to the contact confirmation page but will help if needed @rosariodiaferia-moj